test/index.js

```diff
@@ -0,0 +1,33 @@
+const path = require('path');
+const expect = require('chai').expect;
+const jsdom = require('jsdom');
+
+describe('node acceptance tests', function() {
+  let _window;
+
+  beforeEach(function() {
+    _window = global.window = jsdom.jsdom().defaultView;
+    global.navigator = _window.navigator;
+    global.document = _window.document;
+    Object.defineProperty(_window, 'WebVRConfig', {
+      get() {
+        return global.WebVRConfig;
+      },
+      set(WebVRConfig) {
+        global.WebVRConfig = WebVRConfig;
+      }
+    });
+  });
+
+  afterEach(function() {
+    delete global.window;

```

Comment:
`_window` is a local variable only in this test file. It is not applied to `global`.